README.md

```diff
@@ -79,7 +79,7 @@ The following sections describe how to build and run Cloud Hypervisor.
 ## Host OS
 
 For required KVM functionality the minimum host kernel version is 4.11.  For
```

Comment:
Will CH work with 4.11 after this change? Otherwise this is the version that should be bumped!

Reply:
Well, what's the difference of "required KVM functionality" and "adequate performance" please? I am not sure about this. 
If we update the "required KVM functionality" version to 5.13, then the "adequate performance" version should update to 5.13 together? I suppose so.

Reply:
Well, the older version was the minimum that would work - lower than that it should bail on a failure to turn on one of the required extensions. The problem is that we need an new version due to a bug in the kernel that we can't easily detect - hence why that CPUID bit was disabled.

What is the impact of running with this CPUID bit turned off?

Reply:
> What is the impact of running with this CPUID bit turned off?

When the guest vcpu access memory swapped out by host its execution is suspended until memory is swapped back. Asynchronous page fault is a way to try and use guest vcpu more efficiently by allowing it to execute other tasks while page is brought back into memory[1]

So the performance of guest may be impacted if the APF is disabled.

Though it's difficult to decide kernel version, how about add a option in `--cpus features=` like `amx`? Disable APF by default and turn it on if people need it.

1. https://www.linux-kvm.org/images/a/ac/2010-forum-Async-page-faults.pdf

Reply:
I'd rather not have the option. If we move the minimum supported version what impact will that have on our users i.e. what host distributions will be excluded.

Reply:
Sorry for the late reply.
So, do you mean update both of the minimum and recommended host kernel to 5.13?

Reply:
Sure, lets just do that!